packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -2196,28 +2196,6 @@ describe.each([
           expect(response.rows).toHaveLength(0)
         })
 
-      it("queries the row api passing the view fields only", async () => {
-        const searchSpy = jest.spyOn(sdk.rows, "search")
-
-        const view = await config.api.viewV2.create({
-          tableId: table._id!,
-          name: generator.guid(),
-          schema: {
-            id: { visible: true },
-            one: { visible: false },
-          },
-        })
-
-        await config.api.viewV2.search(view.id, { query: {} })
-        expect(searchSpy).toHaveBeenCalledTimes(1)
-
-        expect(searchSpy).toHaveBeenCalledWith(
-          expect.objectContaining({
-            fields: ["id"],
-          })
-        )
```

Comment:
This was changed so that the ViewV2 is passed down and `fields` is undefined. `sdk.rows.search` has been coded to know that if `fields` is undefined, it derives all of the visible fields from the view. So this test isn't needed anymore.